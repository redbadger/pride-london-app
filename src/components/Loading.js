// @flow
import React from "react";
import Text from "./Text";

const Loading = () => <Text>Loading...</Text>;

export default Loading;
