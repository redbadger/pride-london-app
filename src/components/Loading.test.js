import React from "react";
import { shallow } from "enzyme";
import Loading from "./Loading";

it("renders correctly", () => {
  const output = shallow(<Loading />);
  expect(output).toMatchSnapshot();
});
