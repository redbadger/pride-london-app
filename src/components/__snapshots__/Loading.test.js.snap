// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  color="blackColor"
  markdown={false}
  markdownStyle={Object {}}
  type="text"
>
  Loading...
</Text>
`;
